### DISCUSSION TITLE
Determenism and Network: Positions acumulate error when object hits corner.

### DISCUSSION THREAD

**Discussion by RonYanDaik** [godot-jolt/godot-jolt#794] [Questions]
Didn't know where to post it. Not sure it its a bug or not.
But I've been testing Jolt with networking and had some issues to figure out.
But now when I'm kinda on right path I've stumbled on this thing that I don't know what to do with.

As you can see on video below (sorry for twitter link) simulation on server and client (with latency 150) runs close (0.0 error) until you hit a corner. Here the error starts to accumulate. I'm not sure if I'm doing something or error is somewhere else.

https://twitter.com/YanDaik/status/1760766197354868788

Maybe someone encountered similar thing.

Any ideas?

**Comment by mihe** [comment]
As mentioned in [the readme](https://github.com/godot-jolt/godot-jolt?tab=readme-ov-file#what-about-determinism) of this repository, this extension makes no guarantees about perfect determinism, nor does Godot itself for that matter. Something like this could presumably creep in from pretty much anywhere.
